### PR TITLE
Update karmadactl init output

### PR DESCRIPTION
**Pull request by jwcesign**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Update karmadactl init output

**Which issue(s) this PR fixes**:
Fixes #none

**Special notes for your reviewer**:
@Poor12

### CODE REVIEW

**Comment by Poor12** [comment]
Sync to the zh doc together.

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/installation.md [hunk 1]
```diff
@@ -37,8 +37,10 @@ kubectl karmada init
 ```
 安装好可能需要大约 5 分钟。如果一切正常，你将看到输出类似于：
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/installation.md:38]:
```suggestion
安装过程需要大约 5 分钟。如果一切正常，你将看到类似的输出：
```

**Review by RainbowMango — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**